### PR TITLE
Update ip-addresses.md

### PR DESCRIPTION
**Pull request by xavpaice**
Change https://github.com/replicatedhq/troubleshoot/pull/734 removes IP address redaction from Troubleshoot - This accompanying documentation change should be made when that is released.

### CODE REVIEW

**Review by paigecalvert — DISMISSED** [review]
Did some minor rewording. LMK if it looks okay.

**Comment by xavpaice** [comment]
@paigecalvert thanks for the review and the updates!  Versions move fast, I had to correct the version because we did a release, otherwise this looks great.  Once you're happy, I'm keen to squash/merge.

**Review by paigecalvert — DISMISSED** [review]

**Comment by paigecalvert** [comment]
@xavpaice just re-approved! I'll let you and/or @chris-sanders merge when the release is ready to go out 👍

**Review by chris-sanders — APPROVED** [review]
Looks good.

**Pull request merged**
